setup.py

```diff
@@ -45,8 +45,10 @@
 
 # Headers
 
-pxd_inc_dir = os.path.join(cwd, "src", "vendor", "inc")
-sys.path.insert(0, pxd_inc_dir)
+# Do not use an absolute path in include_dirs, otherwise the build
+# will vary depending on the build path
+pxd_inc_dir = os.path.join(".", "src", "vendor", "inc")
+sys.path.insert(0, os.path.join(cwd, pxd_inc_dir))
```

Comment:
If I remove the differently variables to do a one-line, you are changing:
```python
sys.path.insert(0, os.path.join(os.path.abspath('.'), "src", "vendor", "inc"))
```
to
```python
sys.path.insert(0, os.path.join(os.path.abspath('.'), os.path.join(".", "src", "vendor", "inc")))
```

I don't see the difference?

Reply:
the point is to have a relative pxd_inc_dir so that the build is reproducible, sys.path doesn't matter

Reply:
pxd_inc_dir is included in the generated source code, so if it's absolute it varies with the build path - if it's relative then it's stable

Reply:
ha ok, I was confused on the Git diff, but yeah I realized you focus on the `pxd_inc_dir` usage elsewhere in the file

Reply:
No worries, thanks for the review